dissect/target/plugins/os/windows/search.py

```diff
@@ -0,0 +1,252 @@
+        # Windows 11 22H2 (SQLite3)
+        "sysvol/ProgramData/Microsoft/Search/Data/Applications/Windows/Windows.db",
+        # Windows Vista and Windows 10 (EseDB)
+        "sysvol/ProgramData/Microsoft/Search/Data/Applications/Windows/Windows.edb",
+        # Windows XP (EseDB)
+        "sysvol/Documents and Settings/All Users/Application Data/Microsoft/Search/Data/Applications/Windows/Windows.edb",  # noqa: E501
+    )
+
+    USER_PATHS = (
+        # Windows 10 Server Roaming (EseDB / SQLite)
+        "AppData/Roaming/Microsoft/Search/Data/Applications/S-1-*/*.*db",
+    )
+
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self.databases = set(self.find_databases())
+
+    def find_databases(self) -> Iterator[tuple[Path, UserDetails | None]]:
+        seen = set()
+
+        for system_path in self.SYSTEM_PATHS:
+            if (path := self.target.fs.path(system_path)).is_file() and (
+                digest := hashutil.common(path.open())
+            ) not in seen:
+                seen.add(digest)
+                yield path.resolve(), None
+
+        for user_details in self.target.user_details.all_with_home():
+            for user_path in self.USER_PATHS:
+                for path in user_details.home_path.glob(user_path):
+                    if (digest := hashutil.common(path.open())) not in seen:
+                        seen.add(digest)
+                        yield path.resolve(), user_details
+
+    def check_compatible(self) -> None:
+        if not self.databases:
+            raise UnsupportedPluginError("No Windows Search Index database files found on target")
+
+    @export(record=[SearchIndexRecords])
+    def search(self) -> Iterator[SearchIndexRecords]:
+        """Yield Windows Index Search records.
+
+        Windows Search is a standard component of Windows 7 and Windows Vista, and is enabled by default. The standard (non-Windows Server)
+        configuration of Windows Search indexes the following paths: ``C:\\Users\\*`` and ``C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\*``,
+        with some exceptions for certain file extensions (see the linked resources for more information).
+
+        Parses ``Windows.edb`` EseDB and ``Windows.db`` SQlite3 databases. Currently does not parse
+        ``GatherLogs/SystemIndex/SystemIndex.*.(Crwl|gthr)`` files or ``Windows-gather.db`` and ``Windows-usn.db`` files.
+
+        The difference between the fields ``System_Date*`` and ``System_Document_Date*`` should be researched further.
+        It is unclear what the field ``InvertedOnlyMD5`` is a checksum of (record or file content?). It might be possible
+        to correlate the field ``System_FileOwner`` with a ``UserRecordDescriptor``. The field ``System_FileAttributes`` should be
+        investigated further.
+
+        No test data available for indexed Outlook emails, this plugin might not be able to handle indexed email messages.
+
+        Resources:
```

Comment:
```shell
❯ rg References: dissect | wc -l          
     130

❯ rg Resources: dissect | wc -l 
      77
```
We're not great on either but we should definitely pick a side.

I think `References:` is the one that's supported by Sphinx: https://github.com/sphinx-doc/sphinx/blob/01af04241b1305e7bdb8687c45381c5638275d2f/sphinx/ext/napoleon/docstring.py#L396